### PR TITLE
Front pr

### PR DESCRIPTION
**Pull request by jnougaro**
Front PR new attempt without node_modules and .env files.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (twagger) [inline .prettierrc:1]: Hello, il faudra supprimer ce fichier aussi (c'est un fichier de parametrage de ton vscode), tu peux le git rm et repush sans refaire une PR

**Review by twagger — APPROVED** [review]
Okay pour moi

**Review by fmonbeig — APPROVED** [review]
Good for me

**Review by Mastodoonte — APPROVED** [review]
All good

File: srcs/client/src/Pages/Home/home.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  useEffect(() => {
+    localStorage.getItem("pathIsFree");
+    console.log("test");
+  });
+
+if (!(localStorage.getItem("pathIsFree")))
+{
+    return (
+        <>
```
Comment (lefoun) [inline srcs/client/src/Pages/Home/home.tsx:21]: I think there's some refactoring to be done here these seem to be largely the same

**Review by lefoun — APPROVED** [review]
Some refactoring to be done.
Some code needs to be deleted (for example stale code that isn't being used or commented code).
Some file types are JSX is there a reason for that ? We have to use typescript.

**Pull request merged**